src/slate-editor/slate-editor.tsx

```diff
@@ -42,8 +44,7 @@ const kEmptyEditorValue = textToSlate("");
 const kDefaultHotkeyMap = {
         'mod+b': (editor: Editor) => editor.toggleMark(EFormat.bold),
         'mod+i': (editor: Editor) => editor.toggleMark(EFormat.italic),
-        'mod+u': (editor: Editor) => editor.toggleMark(EFormat.underlined),
-        'mod+\\': (editor: Editor) => editor.toggleMark(EFormat.code)
+        'mod+u': (editor: Editor) => editor.toggleMark(EFormat.underlined)
       };
```

Comment:
Unrelated, but in discussions with others the `code` mark should not be considered a default, and this particular keyboard shortcut (which I basically invented) has even less reason to be considered a global default.